src/table-to-have.ts

```diff
@@ -498,8 +500,13 @@ export const toHaveColumnValuesInSet = (
   targetSet: Set<string>,
 ) => {
   tableData.forEach((row) => {
+    if (!(columnHeader in row)) {
```

Comment:
Nice, I think a set of validators would be a nice idea at some point. Quite possible this error handling will reappear.

Reply:
Good shout, there are a few other validations, like null checks, that appear a few times so they could benefit as well